test-harness/gold-master-files/__path____no_auth_pets_10___method___GET___headers____accept___application_idonotexist___.json

```diff
@@ -12,20 +12,19 @@
     "statusText": "Not Acceptable",
     "headers": {
       "content-type": [
-        "application/problem+json"
+        "application/json; charset=utf-8"
       ],
       "content-length": [
-        "237"
+        "97"
       ],
       "connection": [
         "close"
       ]
     },
     "body": {
-      "type": "https://stoplight.io/prism/errors#NOT_ACCEPTABLE",
-      "title": "The server cannot produce a response matching your content negotiation header",
-      "status": 406,
-      "detail": "Could not find any content that satisfies application/idonotexist"
+      "statusCode": 406,
+      "error": "Not Acceptable",
+      "message": "Allowed: [object Object],application/json"
```

Comment:
What is happening here, is this meant to be object?

Reply:
Very nice catch! The fake regex was missing a `toString()` method, added in 9177b9b9c8f1388babdf6db65557e8fe78d961e2